web_drop_target/static/src/js/web_drop_target.js

```diff
@@ -106,7 +102,42 @@ odoo.define('web_drop_target', function(require) {
                     );
                 }
             });
-        }
+        },
+
+        _handle_file_drop_attach: function(
+            item, e, res_model, res_id, extra_data
+        ) {
+            var self = this;
+            var file = item.getAsFile();
+            var reader = new FileReader();
+            reader.onloadend = self.proxy(
+                _.partial(self._create_attachment, file, reader, e, res_model, res_id, extra_data)
+            );
```

Comment:
Can you please add FileReader error handler ?

Reply:
@fuentes010 ping

Reply:
Done

Reply:
@fuentes010 I think @MeetKD was asking for an overrideable error handler (as `self._create_attachment`).
Once this is done I think we are ready to merge.

Reply:
@MeetKD @simahawk done.